### PR TITLE
Add kubernetes to Travis CI

### PR DESCRIPTION
**Pull request by mlangbehn**
Requisite changes for Kubernetes to run in Travis CI.

List of changes:

* Change mode of quickstart.sh, and quickstart-postgres-svc.sh to 0755
* Move quickstart.sh, and quickstart-postgres-svc.sh to scripts/
* Travis CI now installs tools via scripts/install.sh
* Travis CI now uses BlueMix authentication via scripts/bx_auth.sh
* Travis CI now runs tests/test-kubernetes.sh for all tests not being
  run on a pull request

Signed-off-by: Matt Langbehn <matthew.langbehn@gmail.com>

### CODE REVIEW

**Comment by mlangbehn** [comment]
Do not merge yet. I've almost got this working.

File: .travis.yml [hunk 1]
```diff
@@ -13,20 +13,22 @@ addons:
...
       - shellcheck
 
-install:
-  - pip install -U pip
+matrix:
+  fast_finish: true
+
+env:
+  - TEST_RUN="./tests/test-docker-compose.sh"
+  - TEST_RUN="./tests/test-kubernetes.sh"
```
Comment (markstur) [inline .travis.yml]: setting TEST_RUN twice?

Comment (mlangbehn) [reply .travis.yml]: Yes, this causes Travis to run multiple jobs (one for docker-compose, and one for Kubernetes).

Comment (markstur) [reply .travis.yml]: ok.  Interesting that they went with the same syntax for that as for setting 2 vars. Not your problem though.

**Comment by kennjason** [comment]
LGTM

**Pull request merged**